applications/MultilevelMonteCarloApplication/python_scripts/mc_utilities.py

```diff
@@ -5,249 +5,473 @@
 
 # Import packages
 import numpy as np
+from scipy.stats import norm
 from math import *
+import copy
+
+# Importing the analysis stage classes of the different problems
+from simulation_definition import SimulationScenario
 
 # Import the StatisticalVariable class
-from auxiliary_classes_utilities import StatisticalVariable
+from statistical_variable_utilities import StatisticalVariable
```

Comment:
side note: this way of importing python-scripts is deprecated and will stop working at some point (not soon though) therefore we encourage devs to slowly adapt the new way of importing:
```suggestion
from KratosMultiphysics.MultilevelMonteCarloApplication.statistical_variable_utilities import StatisticalVariable
```

Reply:
Thanks

Reply:
You can check the CompressiblePotentialApp, they already use the [new import system](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CompressiblePotentialFlowApplication/CompressiblePotentialFlowApplication.py#L11)

Reply:
Ty, done!